security-analysis/security-analysis-api/src/main/java/com/powsybl/security/extensions/VoltageExtension.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.extensions;
+import com.powsybl.commons.extensions.AbstractExtension;
+import com.powsybl.security.LimitViolation;
+
+/**
+ * Extension to handle pre-contingency voltage value for a voltage limit violation
+ *
+ * @author Olivier Bretteville <olivier.bretteville at rte-france.com>
+ */
+public class VoltageExtension extends AbstractExtension<LimitViolation> {
+
+    private final float preContingencyValue;
```

Comment:
@sylvlecl Should we use a `float` or a `double` there?

Reply:
Ah, well spotted, we should use `double` as in the network and similar extensions.